src/plugins/intel_npu/src/backend/include/zero_backend.hpp

```diff
@@ -28,6 +28,15 @@ class ZeroEngineBackend final : public IEngineBackend {
     bool isBatchingSupported() const override;
     bool isWorkloadTypeSupported() const override;
 
+    void updateInfo(const Config& config) {
+        _logger.setLevel(config.get<LOG_LEVEL>());
+        if (_devices.size() > 0) {
+            for (auto dev : _devices) {
```

Comment:
`for (auto dev : _devices)`  -> `for (auto & dev : _devices)` to avoid copy?

Reply:
Fix in [new  commit](https://github.com/openvinotoolkit/openvino/pull/25388/commits/bea8f81c57b6d6006855216ecadf81c545c863b7).